airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationResolver.java

```diff
@@ -37,11 +37,12 @@ public static boolean isAzureBlobCopy(final JsonNode config) {
         && config.get("loading_method").has("azure_blob_storage_account_name");
   }
 
-  public static Map<DestinationType, Destination> getTypeToDestination() {
-    final SnowflakeS3StagingDestination s3StagingDestination = new SnowflakeS3StagingDestination();
-    final SnowflakeGcsStagingDestination gcsStagingDestination = new SnowflakeGcsStagingDestination();
-    final SnowflakeInternalStagingDestination internalStagingDestination = new SnowflakeInternalStagingDestination();
-    final SnowflakeCopyAzureBlobStorageDestination azureBlobStorageDestination = new SnowflakeCopyAzureBlobStorageDestination();
+  public static Map<DestinationType, Destination> getTypeToDestination(
+                                                                       final String airbyteEnvironment) {
```

Comment:
this should be on the same line as above, right?

Reply:
For some reason, it started looking line that after "gradlew format" command (this command is also run in CI). I tried to make it a single line manually twice, but the automatic format sets it as you see now. Have no idea why

Reply:
In that case, we can keep this change and it appears you're already in the publication process plus it's not a functional requirement. Thanks for the context though